### PR TITLE
[11.0][ADD] shipping_address_type to ao_base:  #22116

### PR DESCRIPTION
**Pull request by AdriaGForgeFlow**
Adds a field shipping_address_type to res_partner model. Related to [#22116](https://projects.alephobjects.com/projects/oca-12-migration-tickets/work_packages/22116/activity)

cc ~ @Eficent @hveficent

### CODE REVIEW

File: ao_base/models/res_partner.py [hunk 1]
```diff
@@ -1,12 +1,20 @@
 # Copyright 2018 Eficent Business and IT Consulting Services S.L.
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
 
-from odoo import api, models
+from odoo import api, models, fields
 
 
 class ResUsers(models.Model):
```
Comment (HviorForgeFlow) [inline ao_base/models/res_partner.py]: replace ResUsers/ResPartner

File: ao_base/models/res_partner.py [hunk 2]
```diff
@@ -1,12 +1,20 @@
 # Copyright 2018 Eficent Business and IT Consulting Services S.L.
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
 
-from odoo import api, models
+from odoo import api, models, fields
```
Comment (HviorForgeFlow) [inline ao_base/models/res_partner.py]: display imports on alphabetic order

**Review by HviorForgeFlow — CHANGES_REQUESTED** [review]
Rebase required also

**Comment by AdriaGForgeFlow** [comment]
Closing, new PR here: #212

**Pull request closed without merging**